pytest_odoo.py

```diff
@@ -165,7 +164,7 @@ def load_registry():
     # since Odoo sets it when loading test suites.
     threading.current_thread().testing = True
     with _worker_db_name() as db_name:
-        odoo.registry(db_name)
+        odoo.modules.registry.Registry(db_name)
```

Comment:
This  means that we won't be able to use the latest releases w/ odoo < 18.
Can't we leverage an ImportError instead?
If not, fine, but we have to document that this breaks compat w/ former odoo versions.

Reply:
It seems to work with < 18

Reply:
tested with version 16.0

Reply:
Great! Thanks for your feedback!